x-pack/legacy/plugins/canvas/README.md

```diff
@@ -99,13 +97,6 @@ const browserFunctions = [
     }
   }),
 ];
-
-// Register our elements and browserFunctions with the Canvas interpreter.
-kbnInterpreter.register({
```

Comment:
Need to keep this as it's still the way for another plugin to add to Canvas.  Hopefully this will change during the next release, but for now, it's the way.

Reply:
Reverted this change.